packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -53,10 +55,29 @@ export abstract class InversifyHttpAdapter<
     return httpAdapterOptions.logger;
   }
 
+  #buildCorsOptions(
+    httpAdapterOptions: HttpAdapterOptions | undefined,
+  ): CorsOptions {
+    if (
+      httpAdapterOptions?.cors === undefined ||
+      typeof httpAdapterOptions.cors === 'boolean'
+    ) {
+      return {
+        methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
+        optionsSuccessStatus: 204,
+        origin: '*',
```

Comment:
I don't know if we want to encourage the use of a wildcard origin, it's considered a bad practice